src/tools/GCLogParser/parse-hb-log.cs

```diff
@@ -603,10 +603,10 @@ static void PrintHeader()
 
             swPassOneFiles[(int)PassOneViewType.Thread].WriteLine("##########Thread view - indices of threads currently running on each proc in between GCs\n");
             swPassOneFiles[(int)PassOneViewType.AllocHeap].WriteLine("##########Alloc heap view - alloc heaps for threads currently running on each proc in between GCs\n");
-            string strHeader = string.Format("{0,6}", "ms");
+            string strHeader = $"{"ms",6}";
```

Comment:
This looks wrong.  Does this actually compile? How are you validating your changes to this file (it's not part of Corelib)?